packages/client/src/runtime/core/tracing/TracingHelper.ts

```diff
@@ -0,0 +1,59 @@
+import type { Context } from '@opentelemetry/api'
+import { EngineSpanEvent, ExtendedSpanOptions, SpanCallback, TracingHelper } from '@prisma/internals'
+
+export const disabledTracingHelper: TracingHelper = {
+  isEnabled() {
+    return false
+  },
+  getTraceParent() {
+    return `00-10-10-00`
+  },
```

Comment:
Where does this "magic string" come from?

Reply:
See comment [here](https://github.com/prisma/prisma/pull/19039/files#diff-bf97d215759ee6d6552a229d2be2ea08ea271db90803c961f29b57438c2ba7bfR20). I should probably duplicate it here as well. Other option would be moving it to `internals`, but that would require making `internals` a proper dependency of `instrumentation` package (rather than `devcDependency`) and that might be too much for one constant